macros/src/actix/mod.rs

```diff
@@ -124,12 +124,16 @@ pub fn emit_v2_definition(attrs: TokenStream, input: TokenStream) -> TokenStream
         Data::Struct(ref s) => match &s.fields {
             Fields::Named(ref f) => handle_field_struct(f, &mut props_gen),
             Fields::Unnamed(ref f) => {
-                let s = f.span().unwrap();
-                s.warning(
-                    "tuple structs do not have named fields and hence will have empty schema.",
-                )
-                .emit();
-                s.help(&*EMPTY_SCHEMA_HELP).emit();
+                if f.unnamed.len() == 1 {
```

Comment:
I assumed we check for `newtype` / 1 tuple structs ?

Reply:
That's correct.